print-apps/oereb/toc.jrxml

```diff
@@ -147,7 +146,7 @@
 				</textElement>
 				<textFieldExpression><![CDATA[$R{TableofContentLabel}]]></textFieldExpression>
 			</textField>
-			<textField isStretchWithOverflow="true" isBlankWhenNull="true">
+			<textField textAdjust="StretchHeight" isBlankWhenNull="true">
```

Comment:
Did you test if this works in MFP?
I think it will breach the report.
It is added automatically by newer versions of JasperSoft studio when you change touch the `TextField`. In my experience it does not work with MFP so just use the old line if it causes an issue.

Reply:
It works here - by "working" I mean the PDF gets created. Should we leave it so then?

Reply:
OK that is great, then I would leave it and keep it in mind ;-). It might depend on the Jasper Report version that MFP is using that now it works or that it is still working.
I think the `textAdjust` is any was the newer attribute so it is better to us this as long as it works.